truffle/src/main/java/org/jruby/truffle/pack/nodes/read/ReadDoubleNode.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
The read nodes are a screen of code each - they're not that tiny. Do you mean the write nodes? They are pretty small.